src/components/shared/AlertBox.tsx

```diff
@@ -72,12 +72,14 @@ const AlertBox = forwardRef<any, Props>(function NavLinkRef(
         [severity, short]
     );
 
+    const iconProps = hideIcon ? { icon: hideIcon } : {};
```

Comment:
The old icon being set to `undefined` was throwing an error due to messed up DOM.